src/Organisms/Modal/Modal.js

```diff
@@ -0,0 +1,39 @@
+// @flow strict
+import PropTypes from 'prop-types'
+import * as React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 100%;
+  height: 100%;
+  background-color: ${(props) => props.theme['modalForegroundColor']};
```

Comment:
use the interpolation helpers (it will also improve the type coverage and fix the build)

Reply:
mm ok !